src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -664,7 +664,8 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     .subscribe(
       resp => {
         this.isLoading = false;
-        this.sendNotification('Editor', 'Deleted: ' + name);
+        this.snackBar.open('Deleted: ' + name,
```

Comment:
This would turn the delete from a Zowe notification into an Editor popup. Is this what we want? Didn't we decide deletes are important now... don't remember what UX liked